lib/tests.nix

```diff
@@ -173,7 +173,7 @@ let
             })
           ];
 
-          boot.initrd.services.swraid.enable = true;
+          boot.swraid.enable = true;
```

Comment:
This also needs to be wrapped like in mdadm.nix. since we export the rest function in the module

Reply:
Done.